.gitignore

```diff
@@ -2,9 +2,10 @@
 __pycache__/
 *.py[cod]
 *$py.class
-
+*.S
```

Comment:
why this change?

Reply:
try to ignore assembly files generated in commands.

Reply:
ah ok. I guess this means we can't check in assembly without removing it, but we can do that later too.